### ISSUE TITLE
Example for logging implementation

### ISSUE THREAD

**Issue by kumaranvpl** [Ayiga/go-kit-middlewarer#4]
Can you provide an example for logging implementation? Also update examples folder.

**Comment by Ayiga** [comment]
Thanks for the feedback.  I assume you mean to just add the logging by default to the example?

I have done that in this case via the commit [f18b206a](https://github.com/Ayiga/go-kit-middlewarer/commit/f18b206a33f21b28116cef73fd9c100db90efbeb)

Please let me know if you have any questions, or if that was not what you meant.

**Comment by kumaranvpl** [comment]
@Ayiga Yeah.. this is what I wanted. It will be much more helpful if you add examples for loadbalancing and client generation too.
